Uplift/Views/Dropdown/DropdownHeaderView.swift

```diff
@@ -41,10 +45,22 @@ class DropdownHeaderView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
-    @objc func didTapView() {
+    func rotateArrowDown() {
+        UIView.animate(withDuration: 0.3) {
+            self.arrowImageView.transform = CGAffineTransform(rotationAngle: .pi/2)
+        }
+        isArrowRotated = true
+    }
+
+    func rotateArrowUp() {
         UIView.animate(withDuration: 0.3) {
-            self.arrowImageView.transform = CGAffineTransform(rotationAngle: self.isArrowRotated ? 0 : .pi/2)
+            self.arrowImageView.transform = CGAffineTransform(rotationAngle: 0)
```

Comment:
If this is a drop down chevron type thing, if you want it to rotate the opposite way to its original orientation then the trick is to do `CGAffineTransform(rotationAngle: -2 * .pi)`.